docs/source/usage.rst

```diff
@@ -8,130 +8,19 @@ To use perceval, you can directly install the pip wheel in a virtual environment
 
 .. code-block:: bash
 
-   (.venv) $ pip install perceval-quandela
+   (venv) $ pip install perceval-quandela
 
 Alternatively, if you are interested in contributing to the project - you can clone the project from github:
 
 .. code-block:: bash
 
-   (.venv) $ git clone https://github.com/quandela/Perceval
-   (.venv) $ cd Perceval
-   (.venv) $ python setup.py install # or develop for developers
+   (venv) $ git clone https://github.com/quandela/Perceval
+   (venv) $ cd Perceval
+   (venv) $ python setup.py install # or develop for developers
```

Comment:
Can you also change `python setup.py install` into `pip install .` here and everywhere in this repo